tests/bgp/test_bgp_speaker.py

```diff
@@ -115,23 +137,27 @@ def common_setup_teardown(duthost, ptfhost, localhost):
 
     logging.info("########### Done setup for bgp speaker testing ###########")
 
-    yield ptfip, mg_facts, interface_facts, vlan_ips, speaker_ips, port_num, http_ready
+    yield ptfip, mg_facts, interface_facts, vlan_ips, vlan_if_name, nexthops_ipv6, speaker_ips, port_num, http_ready
 
     logging.info("########### Teardown for bgp speaker testing ###########")
 
     for i in range(0, 3):
         ptfhost.exabgp(name="bgps%d" % i, state="absent")
+    logging.info("exabgp stopped")
+
+    for nh in nexthops_ipv6:
+        duthost.command("ip -6 route flush %s/64" % nh.ip)
+    logging.info("Flushed ipv6 nexthop routes from dut")
 
     for ip in vlan_ips:
         duthost.command("ip route flush %s/32" % ip.ip, module_ignore_errors=True)
 
     logging.info("########### Done teardown for bgp speaker testing ###########")
 
-
 def test_bgp_speaker_bgp_sessions(common_setup_teardown, duthost, ptfhost, collect_techsupport):
     """Setup bgp speaker on T0 topology and verify bgp sessions are established
     """
-    ptfip, mg_facts, interface_facts, vlan_ips, speaker_ips, port_num, http_ready = common_setup_teardown
+    ptfip, mg_facts, interface_facts, vlan_ips, _, __, speaker_ips, port_num, http_ready = common_setup_teardown
```

Comment:
should be signle underscore here.

Reply:
I make them different, because some static analysis tools will show an error of overwriting variable with name '_'.
I can fix that.